src/UI/Art/SpriteArtist/Sprite.java

```diff
@@ -182,4 +182,29 @@ public void convertToMask(Color maskColor)
       }
     }
   }
+
+  public void convertToInverseBrightnessMask(Color maskColor)
+  {
+    for( BufferedImage bi : spriteImages )
+    {
+      WritableRaster raster = bi.getRaster();
+      double mask[] = {maskColor.getRed(), maskColor.getGreen(), maskColor.getBlue(), maskColor.getAlpha() };
+      for( int x = 0; x < bi.getWidth(); ++x )
+      {
+        for( int y = 0; y < bi.getHeight(); ++y )
+        {
+          double[] pixel = new double[4];
+          raster.getPixel(x, y, pixel);
+          if( pixel[3] == 0 )
+            continue;
+          double scaledBrightness = (pixel[0] + pixel[1] + pixel[2]) / 3;
+          scaledBrightness = Math.min(255, scaledBrightness * 1.5);
```

Comment:
Why divide by 3 and then multiply by 1.5 instead of just multiplying by 0.5?

Reply:
Clarity.
I'm grabbing an averaged value, then I'm scaling it up - they're semantically different.